### PR TITLE
BDOC-TM dokumentides uuendada viidet BDOC spetsifikatsioonile

### PR DESCRIPTION
**Pull request by olegprokofjev**
IB-4521
Signed-off-by: Oleg Prokofjev <oleg@aktors.ee>

### CODE REVIEW

File: src/SignatureXAdES_B.cpp [hunk 1]
```diff
@@ -66,7 +66,7 @@ const string SignatureXAdES_B::ASIC_NAMESPACE = "http://uri.etsi.org/02918/v1.2.
 const map<string,SignatureXAdES_B::Policy> SignatureXAdES_B::policylist = {
     {"urn:oid:1.3.6.1.4.1.10015.1000.3.2.1",{
         "BDOC – FORMAT FOR DIGITAL SIGNATURES",
-        "https://www.sk.ee/repository/bdoc-spec21.pdf",
+        "http://id.ee/public/bdoc-spec212-eng.pdf",
         // SHA-1
         {   0x80,0x81,0xe2,0x69,0xeb,0x44,0x13,0xde,0x20,0x6e,0x40,0x91,0xca,0x04,0x3d,0x5a,
```
Comment (metsma) [inline src/SignatureXAdES_B.cpp]: You need update the SHA also of the PDF and it create backward compatibility issue

Comment (metsma) [inline src/SignatureXAdES_B.cpp]: You need update the SHA also of the PDF and it creates backward compatibility issue

Comment (olegprokofjev) [inline src/SignatureXAdES_B.cpp]: I've tried to sign with old/new SHA-256. Added new signature (new SHA) to signed file with old signature (old SHA). Tried to open signed file with different application versions (old/new). No problem was noticed (signatures were valid). Am I doing something wrong? Do you mean that IB-4521 should be postponed?

Comment (metsma) [inline src/SignatureXAdES_B.cpp]: Seems that digest verification is disabled currently
https://github.com/open-eid/libdigidocpp/commit/b764a0bfa0e76b8501b1da4efbe34e2af7d158ec
Don't know how digidoc4j and jdigidoc will handle digest verification

**Comment by metsma** [comment]
Telga can you "Squash and merge" this?

**Pull request merged**